Data/Sys/GameSettings/SX4.ini

```diff
@@ -18,3 +18,6 @@ EmulationIssues = The game randomly freezes.
 # Add action replay cheats here.
 
 [Video_Hacks]
+
+[Video_Hacks]
```

Comment:
This line should not be required, as there is already a `[Video_Hacks]` just above

Reply:
Whoops, copy-paste bug.  I'll fix it, thanks!